docs/variables.md

```diff
@@ -184,6 +184,16 @@ Only available for `successCondition`
 | `response.body` | Response body (`string`) |
 | `response.headers` | Response headers (`map[string][]string`) |
 
+### CronWorkflows
+
+> v3.6 and after
+
+| Variable | Description|
+|----------|------------|
+| `cronworkflow.name` | Name of the CronWorkflow (`string`)
+| `cronworkflow.hasLastScheduledTime` | When `cronworkflows.lastScheduledTime` is a sane value this is set to `false` else `true` (`bool`) |
```

Comment:
This description looks like it is inverted from what I'd hope this variable contains.

Reply:
Yes this missed the other changes I suggested [above](https://github.com/argoproj/argo-workflows/pull/13474#discussion_r1721277481) too

```suggestion
| `cronworkflow.hasLastScheduledTime` | `true` if `cronworkflows.lastScheduledTime` exists else `false` (`bool`) |
```